src/app/components/sidebar/sidebar.controller.js

```diff
@@ -9,39 +9,41 @@ angular.module('transmartBaseUi')
     .controller('SidebarCtrl', ['$scope', 'StudyListService', 'EndpointService',
         function ($scope, StudyListService, EndpointService) {
 
-            $scope.publicStudies = [];
-            $scope.privateStudies = [];
+            var ws = this;
```

Comment:
I saw a lot examples use a consistent 'vm' short from View Model when it comes to naming the controller as.  Also as described by this guy

https://johnpapa.net/angularjss-controller-as-and-the-vm-variable/

Shall we do it like that as well?